recipes/pygsheets/meta.yaml

```diff
@@ -0,0 +1,45 @@
+{% set name = "pygsheets" %}
```

Comment:
Can you please use grayskull to generate the recipe? It will make the review process much easier on us.

```
grayskull pypi --strict-conda-forge pygsheets
```

Reply:
Yes, thanks for the review.

Reply:
Done.